docs/apis/calcite.md

```diff
@@ -0,0 +1,529 @@
+Apache Apex,unified stream processing engine enables application developers to process data at very high throughput with low latency. Though type of data demands different type of
```

Comment:
Though type of data demands different type of processing requirements, the SQL remains way to process input data